code/modules/mob/living/living.dm

```diff
@@ -282,6 +282,8 @@
 
 
 /mob/living/ex_act(severity)
+	if(status_flags & GODMODE)
+		return
```

Comment:
bit out of scope but /shrug

Reply:
Yeah since this wasn't really the problem anyway, I can just remove this.